### PR TITLE
HCIDOCS-187: Document procedure for changing BMC address after installation

### PR DESCRIPTION
**Pull request by johnwilkins**
Added a module for editing the baremetalhost resource. This PR replaces https://github.com/openshift/openshift-docs/pull/76624, for which my index got borked. It has passed QE and peer review.

Fixes: [HCIDOCS-187](https://issues.redhat.com//browse/HCIDOCS-187)

See https://issues.redhat.com/browse/HCIDOCS-187 for additional details.

Preview URL: https://77276--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#bmc-addressing_ipi-install-installation-workflow and https://77276--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/bare-metal-configuration.html#editing-a-baremetalhost-resource_post-install-bare-metal-configuration

For release(s): 4.16
QE Review: 

- [x] QE has approved this change. 

Signed-off-by: John Wilkins <jowilkin@redhat.com>

### CODE REVIEW

File: modules/bmo-editing-a-baremetalhost-resource.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// This module is included in the following assemblies: 
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="editing-a-baremetalhost-resource_{context}"]
+= Editing a BareMetalHost resource
+
+After deploying a cluster, you might need to edit a node's `BareMetalHost` resource. For example, if you deploy a cluster with the {ai-full} and you need to add the baseboard management controller (BMC) host name or IP address, or you want to move a node from one cluster to another without deprovisioning it, you must edit the node's `BareMetalHost` resource. To edit a node's `BareMetalHost` resource without deprovisioning the node, use the following procedure.
```
Comment (MahnoorAsghar) [inline modules/bmo-editing-a-baremetalhost-resource.adoc:9]: Carrying the BMC hostname discussion here: Would that not be the domain name like google.com (and is that not different from what hostname means?)

**Comment by ShaunaDiaz** [comment]
@johnwilkins Please resolve merge conflicts and get peer review before merge review.

**Comment by MahnoorAsghar** [comment]
LGTM

**Comment by johnwilkins** [comment]
There are no substantive changes for peer review. I just had to resolve a merge conflict.

File: modules/bmo-editing-a-baremetalhost-resource.adoc (hunk 1 above)
Comment (MahnoorAsghar) [reply modules/bmo-editing-a-baremetalhost-resource.adoc:9]: Makes sense, thanks for the explanation!

**Comment by johnwilkins** [comment]
This was conflicting with HCIDOCS-201.

**Pull request closed without merging**